### PR TITLE
Flake8 fixes and updated installation command

### PR DESCRIPTION
**Pull request by desilinguist**
- Update installation command to use `condo-forge` only, and remove `defaults` channel (#275).
- Run `flake8` on the whole codebase (excluding notebooks) (#287).
- Remove `:date:` fields from module docstrings since we haven't been updating them for everything.

### CODE REVIEW

File: doc/config_rsmtool.rst [hunk 1]
```diff
@@ -269,7 +269,7 @@ A list specifying special ETS-only sections to be included into the final report
 
 standardize_features *(Optional)*
 """""""""""""""""""""""""""""""""
-If this option is set to ``false`` features will not be standardized by dividing by the mean and multiplying by the standard deviation. Defaults to ``true``.
+If this option is set to ``false`` features will not be standardized by subtracting the mean and dividing by the standard deviation. Defaults to ``true``.
```
Comment (aloukina) [inline doc/config_rsmtool.rst]: Nice catch!

File: rsmtool/notebooks/summary/evaluation.ipynb [hunk 2]
```diff
@@ -130,7 +130,7 @@
    "source": [
     "#### Association statistics\n",
     "\n",
-    "The table shows the standard association metrics between human scores and machine scores. Note that some evaluations are based on rounded (`Trim-round`) scores computed by first truncating and then rounding the predicted score."
+    "The table shows the standard association metrics between human scores and machine scores. Note that some evaluations are based on rounded (`raw_trim_round`) scores computed by first truncating and then rounding the predicted score."
```
Comment (aloukina) [inline rsmtool/notebooks/summary/evaluation.ipynb]: There are `raw_trim_round` and `scale_trim_round` - perhaps ``*_trim_round``?

File: rsmtool/notebooks/true_score_evaluation.ipynb [hunk 3]
```diff
@@ -55,8 +55,8 @@
     "                        \"and double-scored responses, and mean squared error (MSE) and \"\n",
     "                        \"proportional reduction in mean squared error (PRMSE) for \"\n",
     "                        \"predicting a true score with system score. As for other evaluations, \"\n",
-    "                        \"these results are computed on the evaluation set. Trim (bound) scores \"\n",
-    "                        \"are truncated to [{}, {}]. Trim-round scores are computed \"\n",
+    "                        \"these results are computed on the evaluation set. `raw_trim` scores \"\n",
+    "                        \"are truncated to [{}, {}]. `raw_trim_round` scores are computed \"\n",
     "                        \"by first truncating and then rounding the predicted score. Scaled scores \"\n",
```
Comment (aloukina) [inline rsmtool/notebooks/true_score_evaluation.ipynb]: Same as above: we have both raw_ and scale_

File: rsmtool/preprocessor.py [hunk 4]
```diff
@@ -2475,7 +2474,7 @@ def process_data_rsmpredict(self, config_obj, data_container_obj):
         h1_mean = df_postproc_params['h1_mean'].values[0]
         h1_sd = df_postproc_params['h1_sd'].values[0]
 
-        #if the we are using a newly trained model, use trim_tolerance from the
+        # if the we are using a newly trained model, use trim_tolerance from the
```
Comment (aloukina) [inline rsmtool/preprocessor.py]:
```suggestion
        # if we are using a newly trained model, use trim_tolerance from the
```

File: rsmtool/utils.py [hunk 5]
```diff
@@ -735,8 +733,25 @@ def difference_of_standardized_means(y_true_observed,
...
     """
-    Calculate the quadratic weighted Kappa
-    in the discrete or continuous cases.
+    Calculate Quadratic-weighted Kappa that works
+    for both discrete and continuous values.
+
+    The formula to compute quadratic-weighted kappa
+    for continuous values was developed at ETS by
+    Matt Johnson (mjohnson@ets.org) and Dan McCaffrey
+    (dmccaffrey@ets.org). The discrete case is simply
```
Comment (aloukina) [inline rsmtool/utils.py]: Wrote to Dan and Matt to confirm.

File: rsmtool/utils.py [hunk 6]
```diff
@@ -735,8 +733,25 @@ def difference_of_standardized_means(y_true_observed,
...
 def quadratic_weighted_kappa(y_true_observed, y_pred, ddof=1):
     """
-    Calculate the quadratic weighted Kappa
-    in the discrete or continuous cases.
+    Calculate Quadratic-weighted Kappa that works
+    for both discrete and continuous values.
+
+    The formula to compute quadratic-weighted kappa
+    for continuous values was developed at ETS by
+    Matt Johnson (mjohnson@ets.org) and Dan McCaffrey
```
Comment (aloukina) [inline rsmtool/utils.py]:
```suggestion
    Shelby Haberman. See `Haberman (2019)  <https://onlinelibrary.wiley.com/action/showCitFormats?doi=10.1002%2Fets2.12258>`_.
```

File: rsmtool/utils.py (hunk 5 above)
Comment (aloukina) [inline rsmtool/utils.py]:
```suggestion
    for full derivation. The discrete case is simply
```

File: rsmtool/notebooks/true_score_evaluation.ipynb (hunk 3 above)
Comment (desilinguist) [reply rsmtool/notebooks/true_score_evaluation.ipynb]: This talks about the scaled scores separately in the next sentence so we don't need to change this.

**Pull request merged**